<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loop2</class>
 <widget class="QDialog" name="Loop2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reactor Loop 2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="layoutLoop2Flow">
       <item>
        <widget class="QLabel" name="labelLoop2Flow">
         <property name="text">
          <string>Flow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineLoop2Flow"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupLoop2Pump">
       <property name="title">
        <string>Recirculation Pumps</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <widget class="QPushButton" name="buttonLoop2Pump2">
          <property name="text">
           <string>Pump 2</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonLoop2Pump1">
          <property name="text">
           <string>Pump 1</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonLoop2Pump3">
          <property name="text">
           <string>Pump 3</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupLoop2Inlet">
       <property name="title">
        <string>Inlet Valves</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="1">
         <widget class="QPushButton" name="buttonLoop2Inlet2Open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonLoop2Inlet3Open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineLoop2Inlet1"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineLoop2Inlet3"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonLoop2Inlet1Open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineLoop2Inlet2"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="buttonLoop2Inlet1Close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buttonLoop2Inlet2Close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="buttonLoop2Inlet3Close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupLoop2Outlet">
       <property name="title">
        <string>Outlet Valves</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineLoop2Outlet1"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineLoop2Outlet3"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonLoop2Outlet1Open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineLoop2Outlet2"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonLoop2Outlet3Open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="buttonLoop2Outlet2Open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="buttonLoop2Outlet1Close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buttonLoop2Outlet2Close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="buttonLoop2Outlet3Close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
